core/development/ui/Events.ts

```diff
@@ -15,11 +16,16 @@ import {
   OVERWORLD_BOARD_ID,
 } from '../Constants';
 import { HEAVEN_BOARD, HELL_BOARD, OVERWORLD_BOARD } from './BoardManager';
+import { hideUnicornAttackButton } from './Screen';
 
 const MOVEMENT_TO_CLICK_THRESHOLD = 10;
 
 let draggedElement: HTMLElement | undefined;
 
+let selectedPieceElement: HTMLElement | undefined = undefined;
+
+let attackingUnicorn: HTMLElement | undefined = undefined;
```

Comment:
Isn't the default value already `undefined`?

Reply:
Yeah, and?

Reply:
Why set it to `undefind`?

Reply:
Cuz im quirky